javacord-core/src/main/java/org/javacord/core/util/rest/RestRequestResultErrorCode.java

```diff
@@ -177,6 +178,7 @@ public enum RestRequestResultErrorCode {
     INVALID_FORM_BODY(50035, "Invalid Form Body"),
     AN_INVITE_WAS_ACCEPTED_TO_A_GUILD_THE_APPLICATIONS_BOT_IS_NOT_IN(
             50036, "An invite was accepted to a guild the application's bot is not in"),
+    INVALID_ACTIVITY_ACTION(50039, "Invalid Activity Action"),
```

Comment:
Seems weird to me that every word start with a capital. Maybe a typo in the docs?
```suggestion
    INVALID_ACTIVITY_ACTION(50039, "Invalid activity action"),
```